ClientApp/src/app/project-page/project-page.component.html

```diff
@@ -30,6 +30,7 @@
       (onChange)="toggleHideCompletedSprints($event)"
     ></p-checkbox>
   </div>
+
```

Comment:
Sorry didn't catch this when we fixed the merge conflicts, can remove this line that was added